components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -357,7 +357,7 @@ function getPageBackground(p: HasImageProps) {
         background-attachment: fixed;
       `};
       background-position: center center;
-      background-image: ${getBackground};
```

Comment:
Hmm, I think this will actually break things as-is because it will override the `background-size`, `background-repeat`, `background-position` and `background-attachment` properties - maybe make this the first thing we set, and then let the other properties override it?

Reply:
Ah, that's right. Moved up 👍